app/assets/stylesheets/_honeycrisp-compact.scss

```diff
@@ -1,6 +1,17 @@
 
 // likely needed in main repo
 .honeycrisp-compact {
+  .container{
```

Comment:
rename to something like .verification-attempts--photo-container  and move for verification-attempst.scss , the .item class can stay the same because you're nesting it and it compiles to .verification-attempts--photo-container .item { 

This will prevent these styles from applying to ANY class .container in all of the hub